test/functional/test-style.js

```diff
@@ -85,6 +85,17 @@ describe('Style', () => {
     expect(st.camelCaseToTitleCase(str)).to.equal('TheQuickBrownFox');
   });
 
+  it('removeAlphaFromColor', () => {
+    expect(st.removeAlphaFromColor('rgba(1, 1, 1, 0)')).to.equal(
+        'rgba(1, 1, 1, 1)');
+    expect(st.removeAlphaFromColor('hsla(1, 1, 1, 0.6)')).to.equal(
+        'hsla(1, 1, 1, 1)');
+    expect(st.removeAlphaFromColor('rgb(1, 1, 1)')).to.equal(
+        'rgb(1, 1, 1)');
+    expect(st.removeAlphaFromColor('hsl(1, 1, 1)')).to.equal(
+        'hsl(1, 1, 1)');
```

Comment:
add tests for percent-based RGBA as well:
hsla(120, 100%, 25%, 0.3)

and maybe some negative cases like:

'#333'
'transparent'
''
'white'
'totallyInvalid'